Userland/Applications/CrashReporter/CrashReporterWindow.gml

```diff
@@ -82,6 +82,13 @@
 
         layout: @GUI::HorizontalBoxLayout
 
+        @GUI::Button {
+            name: "inspect_button"
+            text: "Inspect in HackStudio"
```

Comment:
```suggestion
            text: "Inspect in Hack Studio"
```

https://github.com/SerenityOS/serenity/blob/4b34a1302bf748e45fc17febdbf71088e45e8b5b/Base/res/apps/HackStudio.af#L2

Reply:
Should then also be reflected in the commit message :^)